lib/parser/ast/node.rb

```diff
@@ -34,6 +34,23 @@ def assign_properties(properties)
           @location = location
         end
       end
+
+      POSTFIX_TYPES = Set[:if, :while, :while_post, :until, :until_post].freeze
+      ##
+      # Returns children with an expression, in the order they appear in
+      # the source
+      #
+      def children_in_source_order
```

Comment:
could you mark it as `@private` in YARD please? I don't think anyone except `parser` needs it.

Reply:
Sure. Probably best I isolate it elsewhere